### ISSUE TITLE
Admin user not created during installation

### ISSUE THREAD

**Issue by evgeniy-pupkov** [nkinder/ansible-keycloak#18]
Hello!

Keycloak version: 7.0.0

After deployment using this role, all steps marked as successful. But welcome screen of Keycloak shows that no admin user created and I have to connect from cli to create one. Also, logs contain these lines:
```
2019-09-19 13:22:54,196 INFO  [org.keycloak.services] (ServerService Thread Pool -- 61) KC-SERVICES0050: Initializing master realm
2019-09-19 13:22:56,403 INFO  [org.keycloak.services] (ServerService Thread Pool -- 61) KC-SERVICES0006: Importing users from '/opt/keycloak/keycloak-7.0.0/standalone/configuration/keycloak-add-user.json'
2019-09-19 13:22:56,603 ERROR [org.keycloak.services] (ServerService Thread Pool -- 61) KC-SERVICES0008: Failed to add user  keyadmin to realm  master: realm not found
2019-09-19 13:22:56,611 ERROR [org.keycloak.services] (ServerService Thread Pool -- 61) KC-SERVICES0011: Failed to add user ' keyadmin' to realm ' master': java.lang.NullPointerException
```
As you can see, user name and realm name contain spaces. 
I've overcome it changing 
```
  - -r master
  - -u {{ keycloak_admin_user }}
  - -p {{ keycloak_admin_password }}
```
to
```
- -rmaster
- -u{{ keycloak_admin_user }}
- -p{{ keycloak_admin_password }}
```
in "create Keycloak admin user" step

**Comment by nkinder** [comment]
Thanks for reporting the issue!  Do you mind submitting a PR for this?

**Comment by nkinder** [comment]
This is resolved by commit b5859343a72c35b2457b1f88d4c5d092bd3131e0.  Thanks for the contribution @evgeniy-pupkov !

**Comment by waza-ari** [comment]
@nkinder thanks for merging! Can you please release a new version here such that these fixes are included in new installs via ansible-galaxy?

Thank you very much!
